spec/system/distribution_system_spec.rb

```diff
@@ -306,6 +306,15 @@
     expect(page).to have_css('.alert.error', text: /storage location/i)
   end
 
+  # 4438- Bug Fix
+  it "add item button gets activated when storage location is selected after rendering again on errors" do
```

Comment:
Do we need a separate test for this? Can we just add this assertion to an existing test? System tests are expensive.

Reply:
added to the existing test